src/Appwrite/Realtime/Realtime.php

```diff
@@ -0,0 +1,193 @@
+<?php
```

Comment:
I wondering if it will make sense to combine both Realtime classes into one class. I don't see any major concept differentiation between them.